com.unity.ml-agents.extensions/Runtime/Input/Adaptors/ButtonInputActionAdaptor.cs

```diff
@@ -0,0 +1,45 @@
+#if MLA_INPUT_SYSTEM && UNITY_2019_4_OR_NEWER
+using Unity.MLAgents.Actuators;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.LowLevel;
+
+namespace Unity.MLAgents.Extensions.Input
+{
+    /// <summary>
+    /// Class that translates data between the a <see cref="UnityEngine.InputSystem.Controls.ButtonControl"/> and
```

Comment:
Not a huge deal since we don't generate docs for the extensions package, but you might need to link directly to the web page for the package docs.

Reply:
ah